wklink/links.txt

```diff
@@ -0,0 +1,5 @@
+4871:https://demo.webknossos.org/annotations/Explorational/5b55dbaa1c0000bd008c0e80/readOnly#5149,653,57,0,2.42
```

Comment:
I guess this is just for testing purposes, and we keep track of the links in our infrastructure, right?

Reply:
Yeah, I'll put in different links for testing. The real stuff is in kube